resources/sass/components/_sidebar.scss

```diff
@@ -14,6 +14,21 @@
     }
 }
 .sidebar-wrapper {
+
+    scrollbar-color: #435ebe transparent;
+    scrollbar-width: thin;
+    &::-webkit-scrollbar {
+        width: 4px;
+    }
+    &::-webkit-scrollbar-thumb {
+        background-color: #435ebe;
```

Comment:
Can you change the color of the scrollbar to a neutral one (grayish with opacity)?
And I guess it will be better if the hex value is used it as a variable